test/evergreen.yml

```diff
@@ -1586,6 +1586,27 @@ tasks:
           display_name: Coverage report
           remote_file: wiredtiger/${build_variant}/${revision}/coverage_report/coverage_report_${build_id}.html
 
+  - name: test-wtperf
+    commands:
+      - func: "get project"
+      - func: "compile wiredtiger"
+        vars:
+          posix_configure_flags: --enable-strict --enable-diagnostic
+      - command: shell.exec
+        params:
+          working_dir: "wiredtiger/build_posix"
+          script: |
+            set -o errexit
+            set -o verbose
+
+            mkdir WT_TEST
+            dir=../bench/wtperf/stress
+            for file in `ls $dir`
+            do
+              ./bench/wtperf/wtperf -O $dir/$file -o verbose=2
+              rm -rf WT_TEST/*
```

Comment:
Can we change the code not overwriting WT_TEST?

Reply:
We should not remove the test output in evergreen.

Reply:
After rethinking this, I agree the test output could be useful for troubleshooting when the test fails. 
Even though Jenkins did not save the directory, we can simply do it by replacing this line with `cp WT_TEST WT_TEST_$file`.